snapshot/git/repo/repo.go

```diff
@@ -1,3 +1,6 @@
+// Pacakge repo provides utilities for operating on a git repo.
```

Comment:
Pacakge -> Package.

It would be nice if our code editors caught this for us... I'm pretty sure emacs can spellcheck comments but I haven't ever bothered to set that up.